Fix for add hover effect in Team UI - #169

karagudisuma opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->
Changes for increased scalability for image on hover on team UI, show  social media icon, change the links for the icons accordingly.

## Description
<!--- Describe your changes in detail  -->
When mouse is hovered on the cards of the /aboutus link, the card is scaled 1.6 times and the social media icons are displayed. On mouse out event they are returned to original size without social media icons.
The links to the social media icons are given and tested. 

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->
Solves issue described in #169.

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->
UI changes are tested in localhost. There are no functionality change.

## Screenshots (if appropriate):
![169change](https://user-images.githubusercontent.com/13192124/51475505-82579580-1da8-11e9-84f3-c9f6424b5a90.jpg)


# Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [X ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.

Paarmita commented on src/ignitus-CoreTeam/Styles/style.scss:
```diff
@@ -24,7 +24,22 @@
 
 #team-avatar-card:hover {
-    transform: scale(1.2);
+    transform: scale(1.6);
```
Please change it to 1.2 only as it looks too zoomed out at 1.6.

Paarmita commented on src/ignitus-CoreTeam/Styles/style.scss:
```diff
@@ -24,7 +24,22 @@
     transition-duration: 500ms;
     cursor: pointer;
+    z-index: 9999;
+    position: relative;
```
And position should be absolute. As relative disturbs the UI on hover.

Paarmita commented on src/ignitus-CoreTeam/Components/Team.js:
```diff
@@ -8,23 +8,47 @@ import loader from "../../ignitus-Assets/Images/loader.gif";
           <div className="title-name mt-2">{item.title}</div>
           <div className="title-description">{item.description}</div>
+          <div id={`member-${index}`} class="display-none">
+            {item.facebook && <a href={item.facebook} className="team-sm-icon-color"><i class="fa fa-facebook-square"></i></a>}
```
Social links we need to mention is only angellist and LinkedIn for all members. Can you please remove facebook and replace it with angellist along with Linkedin.

Paarmita commented:
@karagudisuma Thanks for the PR. Can you please also look into this behavior of flex on hover like it comes above the navbar?

![5qyec-z6248](https://user-images.githubusercontent.com/26546816/51486305-5d721b00-1dc6-11e9-8db8-d75198c641a7.gif)

karagudisuma commented:
@Paarmita, @divyanshu-rawat  I have addressed the issues you mentioned above. The screen looks like this now:
Let me know if there is any thing I missed. 
![navbar](https://user-images.githubusercontent.com/13192124/51489493-7d0d4180-1dce-11e9-875c-55199cbc5fa2.jpeg)

-Thanks,
  Suma

Paarmita commented:
@karagudisuma  :+1: This PR looks great - it's ready to merge! :shipit:
